### ISSUE TITLE
预测时，提示Attention Mask 未设置和Attention Mask 未设置

### ISSUE THREAD

**Issue by huangrs494** [shibing624/MedicalGPT#398]
### Describe the Question
Please provide a clear and concise description of what the question is.
实验增量预训练，接入自己的数据进行训练。
1、之后合并模型，使用python inference.py（命令加入了--template_name qwen参数）进行评估出现如下报错：
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:151643 for open-end generation.
2、评估时模型除了输出回复后，还额外输出好多多余英文的信息，而且每次多余的信息内容都是一样的。
![image](https://github.com/user-attachments/assets/42299fec-2196-4ea4-b599-55dda9727f66)

**Comment by shibing624** [comment]
SFT，并加入eos_token

**Comment by shibing624** [comment]
增量预训练，该阶段模型没有对话能力。
